### ISSUE TITLE
waitress notification progress bar position is not responsive(fixed)

### ISSUE THREAD

**Issue by lz100** [JohnCoene/waiter#54]
The progress bar is fixed at a certain window location. It is not responsive. If I scroll or enlarge the window, the progress bar is not moving with the notification bar. 

![not_moving](https://i.ibb.co/HddmsqC/Screenshot-from-2020-06-01-19-23-31.png)

I inspected the html and it comfirms:
![html](https://i.ibb.co/HF38qWx/sadasd.png)
You can see the left, top of notification div is set to `auto` but the progress bar div is set to a fixed number

This is my code to start waitress
```r
    progress <- Waitress$new(theme = "overlay-opacity",
                             min = 0, max =10)
    progress$notify(
        html = "Validating",
        position = "br",
        text_color = "blue"
        )
# I have some other code to update the loop
# ...
```
Is this a bug?

**Comment by lz100** [comment]
I  tested with your waitress [demo](https://shiny.john-coene.com/waiter/) and found the same thing. I think it is not special to my code. 
![demo_screenshot](https://i.ibb.co/m6dCJxq/Screenshot-from-2020-06-01-19-45-50.png)

I can see the progress bar goes off if I scroll the page when it is running.

**Comment by JohnCoene** [comment]
Indeed, I had not noticed, thank you for reporting this, I'll look into it.

**Comment by lz100** [comment]
I made some other tests and found that waitress on button has the same issue but a little different. 
1. It is not responsive, if I change the window size while progress bar is running, it has the same issue as above.
2. Progress bar is set to the highest z-index and it has a separate div that is on top of everything. That means the bar is not inside the parent element containing the button. If I have a collapsable div, when it is collapsed, you would still see the progress bar.
![example](https://i.ibb.co/GCCh5VB/Screenshot-from-2020-06-02-22-27-05.png)
After collapse
![collapse](https://i.ibb.co/7VXNWZ8/Screenshot-from-2020-06-02-22-27-16.png)
